docs/SNMP_Alarms.md

```diff
@@ -41,3 +41,13 @@ obtain the brief description, detailed description, and severity for the alarm:
 ## Alarm Models
 
 The alarm models used by Clearwater are defined in [alarmdefinition.h](https://github.com/Metaswitch/cpp-common/blob/master/include/alarmdefinition.h).
+
+## Alarm Resiliency
```

Comment:
You haven't mentioned the active alarm table in this story. I think that, although some EMSes don't support it, in general this is a big part of our resiliency story - the fact that when an EMS fails and comes back, it can read this table and recover without an operator intervening to run the resync command.